src/routes_initializer.coffee

```diff
@@ -74,6 +78,14 @@ class RoutesInitializer
     mappings[action]() for action in includedActions
 
 
+  # Adjust the current scope for scoped routes
+  scopeDirective: (scopeName, scopedRoutes) =>
+    originalScope = @_scope
+    @_scope = "#{@_scope}#{scopeName}"
```

Comment:
How about using an array here instead of and join them with slashes when adding the actual route?